APCAppCore/APCAppCore/DataSubstrate/Model/APCUser+Bridge.m

```diff
@@ -310,6 +309,18 @@ - (void)signInOnCompletion:(void (^)(NSError *))completionBlock
                             self.sharingScope = APCUserConsentSharingScopeNone;
                         }
                     }
+                    
+                    NSArray *dataGroups = responseDictionary[@"dataGroups"];
+                    self.dataGroups = dataGroups;
+                    
+                    // TODO: Handle multiple consent groups with separate sub populations
+                    NSDictionary *consentStatuses = responseDictionary[@"consentStatuses"];
+                    for (id key in consentStatuses) {
+                        NSDictionary *status = [consentStatuses objectForKey:key];
```

Comment:
For now at least, each subpopulation will have at most one "required" consent, so I'd suggest looking for that here and using that subpop guid rather than just getting the last one. (Also I should maybe fix signInOnCompletion to pass the UserSessionInfo object to the completion handler rather than the raw response dictionary.)